api/attribute.proto

```diff
@@ -394,7 +394,7 @@ message MUPType1SessionTransformedRoute {
   google.protobuf.Any rd = 1;
   uint32 prefix_length = 2 [deprecated = true];
   string prefix = 3;
-  uint32 teid = 4;
+  bytes teid = 4;
```

Comment:
Is this change necessary? I prefer to not break the compatibility. Even if unlikely someone would complain about the API breakage due to this change.

Reply:
Thank you for your review.
I understand the compatibility is important. I will revert this change and push unchanged code.

Reply:
I reverted the API change.